src/platforms/unity/configuration/il2cpp.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: IL2CPP Line Numbers
+description: "Learn how the SDK provides line numbers to issues from IL2CPP builds."
+sidebar_order: 10
+---
+
+<Note>
+
+The IL2CPP line number support is available in Beta. Beta features are still in-progress and may have bugs. We recognize the irony.
+
+</Note>
+
+<Note>
+
+The IL2CPP line number support relies on the debug symbol upload to function properly. With the automated debug symbol upload enabled, the Sentry SDK will upload the line mapping through Sentry CLI.
+
+The feature is available from Unity versions `2020.3` and `2021.3` onwards. We are running tests in CI against the current LTS, starting from `2020.3.37f1` and `2021.3.6f1`.
+
+</Note>
+
+Unity provides you the option to select [IL2CPP](https://docs.unity3d.com/Manual/IL2CPP.html) as scripting backend. This leads to C++ code being generated from your compiled C# (IL) code, hence the name `IL to CPP`.
+
+The Sentry SDK provides line number support for IL2CPP builds for the following platforms:
+
+- Desktop: Windows, macOS, Linux
+- Android
+- iOS
+
+When an error happens in your game, from its perspective, that error happens in native code.
```

Comment:
It's odd to talk about the game's perspective like it's a person. Can we rephrase this?

```suggestion
When an error happens in your game, from its perspective, that error happens in native code.
```

Reply:
Sure! I rephrased it.